pandas/formats/format.py

```diff
@@ -2138,6 +2016,31 @@ def formatter(value):
 
         return formatter
 
+    def _get_formatted_data(self):
+        """
+        Returns formatted float values. This is different from
```

Comment:
`get_result_as_array` - I wouldn't know that from the signature (no preceding `_`)